flake.nix

```diff
@@ -29,5 +34,14 @@
           inherit pkgs;
         };
       }
-    );
+    ))
+    //
+    {
+      overlays.default = (final: prev: {
+        yarnBerry = final.callPackage ./yarn.nix {};
```

Comment:
@madjam002 maybe we could rename it to `yarn-berry_4`? I'd like to keep it consistent with how it's used in the other parts of this repo.